apps/condo/domains/resident/schema/FindOrganizationsForAddressService.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * Generated by `createservice resident.FindOrganizationsForAddressService --type queries`
+ */
+
+const { GQLCustomSchema, find } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/resident/access/FindOrganizationsForAddressService')
+const {
+    MAX_RESIDENT_FIND_ORGANIZATIONS_BY_WINDOW_SEC,
+    RESIDENT_FIND_ORGANIZATIONS_WINDOW_SEC,
+} = require('@condo/domains/resident/constants/constants')
+const {
+    findBillingReceiptsForOrganizations,
+    getOrganizationsWithAcquiring,
+    getOrganizationsWithMeters,
+    getBillingInformationForOrganizations,
+} = require('@condo/domains/resident/utils/serverSchema/findOrganizationsForAddress')
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+const { RedisGuard } = require('@condo/domains/user/utils/serverSchema/guards')
+
+const redisGuard = new RedisGuard()
+
+const checkLimits = async (uniqueField) => {
+    await redisGuard.checkCustomLimitCounters(
+        `find-organizations-by-address-${uniqueField}`,
+        RESIDENT_FIND_ORGANIZATIONS_WINDOW_SEC,
+        MAX_RESIDENT_FIND_ORGANIZATIONS_BY_WINDOW_SEC,
+    )
+}
+
+const FindOrganizationsForAddressService = new GQLCustomSchema('FindOrganizationsForAddressService', {
+    types: [
+        {
+            access: true,
+            type: 'input FindOrganizationsForAddressInput { addressKey: String!, unitName: String, unitType: String, tin: String, accountNumber: String }',
```

Comment:
In designs initial search are done by organization name or tin, where is the name?

Reply:
Should naming be also more clean (not like `tin` (exact field), but something like `tinPart` / `tinSearch` or just `search`)

Reply:
To find organization you need to use Organization.getAll - you can put any condition for name and tin there